src/constants/panels.ts

```diff
@@ -0,0 +1,8 @@
+export const ACCESSORY_PANEL_TAB_INPUT = 'INPUT'
+export const ACCESSORY_PANEL_TAB_PROGRAM = 'PROGRAM'
+export const ACCESSORY_PANEL_TAB_YOUR = 'YOUR'
+export const ACCESSORY_PANEL_TAB_TRACKED_ENTITY = 'TRACKED_ENTITY'
```

Comment:
Perhaps this should be a string literal type called `AccessoryPanelTab`?
```suggestion
export type AccessoryPanelTab = 'INPUT' | 'PROGRAM' | 'YOUR' | 'TRACKED_ENTITY'
```

Reply:
Decided to remove these from this PR as they are not relevant.